java-components/build-request-processor/src/main/java/com/redhat/hacbs/container/analyser/deploy/containerregistry/ContainerRegistryDeployer.java

```diff
@@ -81,12 +82,15 @@ public ContainerRegistryDeployer(
         this.insecure = insecure;
         this.prependTag = prependTag;
         this.imageId = imageId;
+        Log.infof("### Token is %s", token);
+
         String fullName = host + (port == 443 ? "" : ":" + port) + "/" + owner + "/" + repository;
         if (!token.isBlank()) {
             if (token.trim().startsWith("{")) {
-                //we assume this is a .dockerconfig file
-                try (var parser = MAPPER.createParser(token)) {
-                    DockerConfig config = parser.readValueAs(DockerConfig.class);
+                try {
+                    //we assume this is a .dockerconfig file
+                    DockerConfigTemplate config = MAPPER.readValue(token, DockerConfigTemplate.class);
```

Comment:
This was just to simplify the code - using Jib there is an class for the json so seemed simpler to use that.